ui/src/components/SchemaForm/Templates/BaseInputTemplate.tsx

```diff
@@ -143,6 +143,7 @@ const BaseInputTemplate = <
               variant="outline"
               size="sm"
               onClick={() => onChange(defaultValue)}
+              disabled={disabled || readonly}
```

Comment:
```suggestion
              disabled={readonly || disabled}
```
I think readOnly should be prioritized. In a readOnly state, everything should definitively be disabled no matter what